.idea/modules.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Your team will decide whether we should remove this file or not. Personally I would remove this.

Reply:
Removed.